pkg/controllers/provisioning/scheduling/topology_test.go

```diff
@@ -400,6 +400,26 @@ var _ = Describe("Topology", func() {
 			}
 			Expect(nodeNames).To(HaveLen(1))
 		})
+		It("should respect minDomains constraints", func() {
+			var minDomains int32 = 4
```

Comment:
I think you'll need this:
```
	if env.Version.Minor() < 27 {
				Skip("min domains is only supported by default on K8s >= 1.27.x")
			}
```

I haven't looked at the logic in detail, but until 1.27 the minDomains is going to be nil regardless of what you set it to once it hits the API server.

Reply:
Are we able to modify our test environment to pass in the feature flag for minDomains? That way we can test for versions of k8s prior to 1.27

Reply:
Yeah, see https://github.com/aws/karpenter-core/blob/32dcbf1816f5f9bab8c130a91c9557c447a46be8/pkg/test/environment.go#L81 for another place we do that.

Reply:
thx! added